### ISSUE TITLE
allthefallen stopped working

### ISSUE THREAD

**Issue by 4Sol** [rickmiron/variado#11]
Hi~
Could you update allthefallen plugin?
The 'none' thing is also there it seems

https://pastebin.com/5uAuc30L

**Comment by rickmiron** [comment]
Download the plugin again, and if it doesn't work it shows the task info

**Comment by 4Sol** [comment]
The same

https://pastebin.com/ZKBuZt3W

But I see that the files wasn't updated ?
![image](https://github.com/rickmiron/variado/assets/52103769/566d1c21-8afa-421f-a1de-e34749fc49e4)

**Comment by 4Sol** [comment]
I used the plugin again, with the new update, but still it doesn't work for some reason 
https://pastebin.com/bmkKf8Zi
Even not counting the files

**Comment by rickmiron** [comment]
works well for me
so I can't run tests and locate the problem you may have.
I will place a plugin with changes, try it and if it fails show me the task
https://github.com/rickmiron/variado/blob/main/allthefallen_4sol.py

**Comment by 4Sol** [comment]
> works well for me so I can't run tests and locate the problem you may have. I will place a plugin with changes, try it and if it fails show me the task https://github.com/rickmiron/variado/blob/main/allthefallen_4sol.py

https://pastebin.com/eVAzM5fS
Not working
Maybe need to restart PC or router? I will do that, but after finish some slow downloads in chrome

**Comment by 4Sol** [comment]
> add points to know where it stops working, then show the task info

Points? Do you mean in the script? How to?

**Comment by 4Sol** [comment]
Ah, here:
https://pastebin.com/2yQFZRJ6

**Comment by rickmiron** [comment]
download the new plugin update
The information you show is from 1.00, the new plugin is 1.01
https://github.com/rickmiron/variado/blob/main/allthefallen_4sol.py

**Comment by 4Sol** [comment]
I did that, but after double checking and repeating plugin reistnal it still did not work (plugin update) becase I was restartng an existing task, so once I deleted the url from hitomi and pasted it again the plugin was updated too, please check:

https://pastebin.com/pNXi5kQu

**Comment by rickmiron** [comment]
I made changes, try it
https://github.com/rickmiron/variado/blob/main/allthefallen_4sol.py

**Comment by 4Sol** [comment]
well
https://pastebin.com/xgmz88NL
![image](https://github.com/rickmiron/variado/assets/52103769/c8067467-220c-4bdc-8888-c31cc269393c)

**Comment by rickmiron** [comment]
hitomi internally uses its own version of python which is 3.8.10

On your PC, use your own Python and do this:

    print('AAA')
    dato = requests.get('https://booru.allthefallen.moe/posts/541534')
    print(dato.text)
    print(dato.status_code)
    print(dato.headers)
    print('BBB')

and check if there are no problems

**Comment by 4Sol** [comment]
Requests are not importing? I've created a .py file with your code
And separated hello world py which works
![image](https://github.com/rickmiron/variado/assets/52103769/c6cc9960-08e3-43b0-b783-67a3e8a1bfb6)

**Comment by rickmiron** [comment]
install the module with this command:
python -m pip install requests

**Comment by 4Sol** [comment]
Installed and also upgraded pip to 24.0, something is off here
https://pastebin.com/iGXvCScL

> install the module with this command in cmd: python -m pip install requests

**Comment by rickmiron** [comment]
Hello, and there are no problems when you enter through the web? Try using a VPN and let's analyze what problem it could be.

**Comment by 4Sol** [comment]
> Hello, and there are no problems when you enter through the web? Try using a VPN and let's analyze what problem it could be.

I can't access the website normally (boru), I use chrome extensions to unblock many websites.
In incognito mode I get this msg trying to open in:
![image](https://github.com/rickmiron/variado/assets/52103769/0f49bcc4-cba1-400c-9b16-5d94fba64a75)

Using hitomi bult in browser:
![image](https://github.com/rickmiron/variado/assets/52103769/13e62508-a5ef-4735-a261-e92630fcad6f)


How to use VPN? I mean I know how to use it in Chrome, but here should I do something in hitomi dwnlder or?

btw it was working fine before, until hitomi was updated (when that issue with "none" arised).

**Comment by rickmiron** [comment]
Try using the previous version that worked to see if it has no problems.
Is it just your PC or is it your internet that is blocked, try using your smartphone.
Check if you use a free DNS like 1.1.1.1 or 8.8.8.8.
And if nothing works, use the Psiphon program, it is a VPN for the entire PC.

**Comment by 4Sol** [comment]
Try using the previous version that worked to see if it has no problems.
No progress

Is it just your PC or is it your internet that is blocked, try using your smartphone.
Using wifi on smartphone and other ISP - the same

Check if you use a free DNS like 1.1.1.1 or 8.8.8.8.
Here? Didn't help
![image](https://github.com/rickmiron/variado/assets/52103769/9917f488-29d3-46f8-9d88-8dbd4ff4e195)
![image](https://github.com/rickmiron/variado/assets/52103769/39939d00-6ce8-44cc-9359-7d15287d5158)

And if nothing works, use the Psiphon program, it is a VPN for the entire PC.

I am not sure if it was working for the entire PC? When I connected it just launched my browser and did not help with hitomi
![image](https://github.com/rickmiron/variado/assets/52103769/3bb58a39-3f61-4ff7-90af-5120bcc6118c)
![image](https://github.com/rickmiron/variado/assets/52103769/8c754caf-c39a-4fdb-a508-dbd39dfbb5f7)

oof

**Comment by rickmiron** [comment]
Apparently it is the internet that has blocked the IP of allthefallen
Did you try using 1.1.1.1 or 8.8.8.8 as dns?
similar to this:
![ipdns](https://github.com/rickmiron/variado/assets/99231649/c899589e-30d7-4307-8d67-3059e504c6bb)

Did the vpn not work? try changing to other countries
![psifon](https://github.com/rickmiron/variado/assets/99231649/522cc9a6-ac8e-419c-9171-65ab494e1240)

**Comment by 4Sol** [comment]
yes, tried both 1111 and 8888,  as per your screenshot,  but didn't work (IP is detecting automatically though)
and for some reason can't connect to any server of psiphon today
